contrib/python/src/python/pants/contrib/python/checks/tasks/checkstyle/file_excluder.py

```diff
@@ -23,6 +23,11 @@ def __init__(self, excludes_path, log):
             pattern, plugins = line.strip().split('::', 2)
             plugins = plugins.split()
 
+            # Remove this in 1.10.0.dev0 when the `pycheck-pep8` is removed in pycodestyle_subsystem.py
+            if 'pep8' in plugins:
+              log.warn("The pep8 check has been renamed to pycodestyle, please update your suppression file: {}".format(excludes_path))
```

Comment:
Can use `deprecated_conditional` to do this: https://github.com/pantsbuild/pants/blob/2e65f46e044ed6c9fce07a240615e7a96d8b7c27/src/python/pants/build_graph/build_file_aliases.py#L44-L50